coding/stylesheets.md

```diff
@@ -210,3 +210,8 @@ developer working with JavaScript, leave these well alone.
 
 The idea is that -- in order to properly separate our concerns, we
 should never have styling and behaviour bound to the same classes.
+
+
+Code Formatting
```

Comment:
I'm impressed that you followed the heading style of the existing document here, although it's at odds with other documents, without feeling the needs to do extra work to normalise things. To keep this PR focused on its mission. :heart:

I do think that `stylesheets.md` should probably be restructured a bit to give it a new top-level heading, 'cos at the moment it kinda looks like the document is purely about BEM. But that shouldn't happen in this PR.